pkg/engine/templates_generated.go

```diff
@@ -39826,7 +39827,7 @@ Set-AzureCNIConfig
 
     $configJson.plugins.AdditionalArgs[1].Value.DestinationPrefix  = $KubeServiceCIDR
 
-    if ($TargetEnvironment -ieq "AzureStackCloud") {
+    if ($IsAzureStack) {
```

Comment:
is this azure stack specific? 
if this is supposed to apply to all custom clouds, then `IsCustomCloudProfile` may be more appropriate.